recipe/meta.yaml

```diff
@@ -27,14 +27,14 @@ build:
 requirements:
   build:
   host:
-    - python  {{ python_min }}
+    - python {{ python_min }}
```

Comment:
@beckermr, do you know why we are so aggresive on the number of spaces?

Reply:
IDK what you mean. Can you explain more?

Reply:
I mean the linter wants one space between `python` and `{{ python_min }}`. Two spaces is not allowed. Is there a particular reason for that?

Reply:
No particular reason no. I coded it up that way. Happy to change it. We do enforce rules like this in other spots (selectors jinja2 vars) but happy to pass on it here.

Reply:
Thanks for the info